app/src/main/java/com/antosik/benchproject/app/common/binding/swipeRefreshLayout/SwipeRefreshLayoutBindingAdapters.kt

```diff
@@ -0,0 +1,11 @@
+package com.antosik.benchproject.app.common.binding.swipeRefreshLayout
+
+import androidx.databinding.BindingAdapter
+import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+
+@BindingAdapter("isRefreshed")
+fun SwipeRefreshLayout.isRefreshed(isRefreshed: Boolean) {
+    if (isRefreshed) {
+        isRefreshing = false
```

Comment:
Is it really needed to do this in Binding Adapter? Couldn't it be done from the VM/model side?

Reply:
I wrote it in this way because when a user runs an app and `UIState` is loading the progress bar from pull to refresh is visible. Therefore to omit this situation I created a binding adapter that stops the swipe progress bar in the Empty or Success state. 

Only the user can start refreshing data after pulling on the top movie list. At this moment I can't find any solution to delete the binding adapter. Probably I can create new live data for refreshing and setting it like false after execution steps in load movies.

Reply:
So leave it for now and add some issue/todo to rethink it later :D

Reply:
Done 😄 #164